templates/python/hgcBiasedGenProcesses_cfi.py

```diff
@@ -0,0 +1,150 @@
+import FWCore.ParameterSet.Config as cms
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.Pythia8CUEP8M1Settings_cfi import *
+from Configuration.Generator.PythiaUEZ2starSettings_cfi import *
+
+def defineProcessGenerator(process,proc='minbias'):
+    
+    """
+    wraps up a couple of interesting processes for HGCal studies
+    minbias (min.bias), hgg (H->gamgam), wqq (W->qq')
+    """
+
+    #MinBias
+    if proc=='minbias':
+        process.generator = cms.EDFilter("Pythia8GeneratorFilter",
```

Comment:
The first lines of the `EDFilter` are always repeated and could be (at least for the Pythia8 processes) only used in one place. Alternatively, since these lines are probably just copied from the genproductions repository (or the central CMSSW one), it would be good to reference (link) the file from which these processes are taken.